cmd/export.go

```diff
@@ -195,14 +199,40 @@ To export everything about a dataset, use the --dataset flag.`,
 		}
 
 		if exportCmdData {
+
```

Comment:
we should use `core.LoadDataset` to get the path, then `core.LookupBody` to get the data of duplicating this code here.